css/styles.css

```diff
@@ -158,3 +162,559 @@ html {
 .social-icons:hover {
   color: #8a2be2;
 }
+
+.work-section {
+  width: 412px;
+  height: 850px;
+  display: grid;
+  grid-template-rows: 1fr 2fr 3fr;
+}
+
+.box-1 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 0;
+  font-size: 20px;
+  color: #1c1a19;
+  border-bottom: 1px solid #212121;
+  width: 327px;
+  height: 107px;
+  margin: 57px auto;
+}
+
+.box-2 {
+  width: 327px;
+  height: 237px;
+  margin: auto auto;
+}
+
+.box-3 {
+  display: flex;
+  flex-direction: column;
+  width: 327px;
+  height: 337px;
+  margin: 21px auto;
+}
+
+.multi-post-heading {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  display: flex;
+  line-height: 44px;
+  align-items: center;
+  color: #091e42;
+}
+
+.multi-post-para {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  margin: 16px auto;
+}
+
```

Comment:
- [ ] The new changes you made have altered the rending of your CSS styling on the webpage. Kindly fix the display.
- [ ] Note that the section highlighted is the course of the break-in CSS, but just to tag your CSS code.

## New display of your changes 
![image](https://user-images.githubusercontent.com/50990925/212402669-13b4f7f0-3591-4a96-81b1-5bc4fe5bffb5.png)

Reply:
![Mobilephoto](https://user-images.githubusercontent.com/116083031/212410532-f5fb7f6f-6451-4fd2-8d50-3591f5568aeb.jpg)

Kindly check your computer its working completely fine for me in desktop view as well as mobile view.

Reply:
![desktop-photo](https://user-images.githubusercontent.com/116083031/212410762-c1ad9865-fe7a-4c41-b5f2-4f68db30bd5c.jpg)